packages/core/src/Stores/notification-store.js

```diff
@@ -60,6 +63,7 @@ export default class NotificationStore extends BaseStore {
                 ) {
                     this.removeNotifications();
                     this.removeAllNotificationMessages();
+                    this.setClientNotifications();
```

Comment:
a previously used `init()` was called before the entire app state update. that's why I have to update notifications here in a reaction when any of the above values from different stores are set.